SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/RelevancyTest/RelevancyTest.cpp

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Improbable Worlds Ltd, All Rights Reserved
 
 #include "RelevancyTest.h"
+#include "GameFramework/PlayerController.h"
```

Comment:
Why do we need this all of the sudden?

Reply:
GetOwner returns an AActor while GetFlowPlayerController returns a PlayerController (which was incomplete)